### ISSUE TITLE
'columns' adds margin-bottom

### ISSUE THREAD

**Issue by pedzed** [jgthms/bulma#2952]
This is about **Bulma** and what I would consider a margin bug.

### Overview of the problem

- This is about the Bulma **CSS framework**
- I'm using Bulma **version** 0.7.5
- My **browser** is Chrome.

### Description

I am playing a bit with Bulma for the first time, but I have negative experiences with the way Bulma handles `columns` spacings. This issue could possibly be solved with a `no-margin` helper (see: #451).

If you have something like this:
```vue
<div class="columns is-gapless">
    <div class="column">
        <TopSection />
    </div>
</div>
<div class="columns is-gapless">
    <div class="column is-one-quarter">
        <SideSection />
    </div>
    <div class="column">
        <MainSection />
    </div>
</div>
```
then an unexpected `margin-bottom` is added as can be seen in the image (see orange):

![image](https://user-images.githubusercontent.com/8146593/82121932-daca5780-9790-11ea-82e5-34ded3ed63be.png)

When `is-gapless` is removed, then too, you have a spacing:

![image](https://user-images.githubusercontent.com/8146593/82122202-6c869480-9792-11ea-9818-61912fa43acc.png)

I can imagine that removing it would break for a lot of users, so implementing #451 would provide a reasonable workaround.

**Comment by pedzed** [comment]
TL;DR: Spacings should be added manually, not assumed. This would also apply to other cases such as heading titles like `h5.title.is-6`.

![image](https://user-images.githubusercontent.com/8146593/82122327-3dbcee00-9793-11ea-9c31-7f6d666bff3b.png)

**Comment by RasdanJames** [comment]
I just created a helper: has-margin-0 and I put it on all my columns, since I don't like the default margins. But you can also use the SASS file to configure it to have no margin by default I believe.

**Comment by drmnk** [comment]
True for me. When creating forms and want make different width inputs - columns is only available tool, but you had to manually disable it so form looks good.

**Comment by drmnk** [comment]
A little css hack that fix columns margin in form for me:
```
form .columns:not(:last-child) {
  margin-bottom: 0;
}
```
